platform/security/filter/security-filter-login/src/main/java/org/codice/ddf/security/filter/login/LoginFilter.java

```diff
@@ -772,6 +770,11 @@ public int getExpirationTime() {
    * @param expirationTime - time in minutes
    */
   public void setExpirationTime(int expirationTime) {
-    this.expirationTime = expirationTime;
+    // Sets expirationTime to the default if the provided value is less than 2
+    if (expirationTime >= 2) {
+      this.expirationTime = expirationTime;
+    } else {
+      this.expirationTime = DEFAULT_EXPIRATION_TIME;
```

Comment:
✏️ Log something here that the configured expiration time is ignored because it's not valid and the default will be used instead